apps/example/webpack.config.js

```
@@ -0,0 +1,21 @@
+/* eslint-disable no-undef */
+const createConfigAsync = require("@expo/webpack-config");
+const path = require("path");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)